package.json

```diff
@@ -35,6 +36,7 @@
     "jest-haste-map": "^24.5.0",
     "jest-junit": "^6.0.0",
     "jest-resolve": "^24.5.0",
+    "jest-when": "^3.5.1",
```

Comment:
Let's have the implementation part in this PR only. As we agreed tests will be added in a separate ticket.